apps/src/templates/account/account-components.module.scss

```diff
@@ -0,0 +1,43 @@
+.bannerContainer {
+  text-align: center;
+}
+
+.iconContainer {
+  display: flex;
+  gap: 18px;
+  justify-content: center;
+  align-items: center;
+  padding-top: 64px;
+}
+
+.iconContainer img {
+  width: 60px;
+  height: 60px;
+}
+
+.titleContainer {
+  margin-top: 24px;
+}
+
+.titleDesc {
+  margin: 10px 0 0 0 !important;
+}
+
+.cardIcon {
+  width: 45px;
+  height: 40px;
+  color: #0093A4;
+}
+
+.cardContent {
+  padding-bottom: 16px;
+}
+
+.button {
+  width: 100%;
+}
+
+// Increase specificity to override base button
+.cardSecondaryButton.cardSecondaryButton {
+  border: 1px solid #292F36
```

Comment:
Nit: I think this can be set using the `type="secondary"` and `color=buttonColors.black` props on the `<Button>` component.

But this element doesn't look like it's in this PR so feel free to ignore!

Reply:
Oh yes you're right thank you!